remix-lib/src/util.js

```diff
@@ -175,6 +175,20 @@ module.exports = {
     return /a165627a7a72305820([0-9a-f]{64})0029$/
   },
 
+  /**
+    * return a regex which extract the swarmhash from the bytecode, from POC 0.3
+    *
+    * @return {RegEx}
+    */
+  swarmHashExtractionPOC3: function () {
+    return /a265627a7a72315820([0-9a-f]{64})64736f6c6343([0-9a-f]{6})0032$/
```

Comment:
isn't here 32 instead of 64?

Reply:
no you are right, the problem was 31 instead of 30 , the hex of 0
 /a265627a7a72305820([0-9a-f]{64})64736f6c6343([0-9a-f]{6})0032$/

Reply:
ignore this the docs where outdated